### PR TITLE
Add contributing info, roadmap and contact information

### PR DESCRIPTION
**Pull request by yiranwang52**
Included most changes from cody, and some feedback from tech writers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,108 +3,143 @@
...
-- Secure. Supports bi-directional TLS between all components for image tags, and bi-directional TLS between image builder and Kraken for all data.
-- Pluggable storage options. It supports S3/HDFS as storage backend, and it's easy to add more options. It can be setup as write-back cache with configurable TTL, so it can survive S3 outages without impacting functionality.
-- Lossless cross cluster replication. Kraken supports rule-based async replication between clusters.
-- Minimal dependency. Other than plugable storage, Kraken only has optional dependency on DNS.
+# Features
 
-Uber has been using Kraken in production since early 2018. In our busiest cluster, Kraken distributes 1 million 0-100MB blobs, 600k 100MB-1G blobs, and 100k 1G+ blobs per day. At its peak production load, Kraken distributes 20K 100MB-1G blobs under 30 sec with ease.
+Following are some highlights of Kraken:
+- **Highly scalable**. It's capable of distributing docker images at > 50% of max download speed
+  limit on every host; Cluster size and image size do not have significant impact on download speed.
```
Comment (codygibb) [inline README.md]: `s/;/.`

File: README.md [hunk 2]
```diff
@@ -3,108 +3,143 @@
...
+  - Supports at least 8k hosts per cluster.
+  - Supports arbitrarily large blobs/layers. We normally limit max size to 20G for best performance.
+- **Highly available**. No component is a single point of failure.
+- **Secure**. Support uploader authentication and data integrity protection through TLS.
+- **Pluggable storage options**. Instead of managing data, Kraken plugs into reliable blob storage
+  options, like S3, HDFS or another registry. The storage interface is simple, and new options
+  are easy to add.
+- **Lossless cross cluster replication**. Kraken supports rule-based async replication between
+  clusters.
+- Minimal dependencies. Other than pluggable storage, Kraken only has an optional dependency on DNS.
```
Comment (codygibb) [inline README.md]: Make this bold?

File: README.md [hunk 3]
```diff
@@ -3,108 +3,143 @@
...
+
+Kraken has been in production at Uber since early 2018. In our busiest cluster, Kraken distributes
+1 million 0-100MB blobs, 600k 100MB-1G blobs, and 100k 1G+ blobs per day. At its peak production
+load, Kraken distributes 20K 100MB-1G blobs in under 30 sec.
+
+Kraken is a P2P-powered Docker registry which focuses on scalability and availability.
```
Comment (codygibb) [inline README.md]: This is repeated at the top of the file, remove from here?

File: README.md [hunk 4]
```diff
@@ -3,108 +3,143 @@
...
+reliable hybrid cloud setup.
 
-On top of that, Kraken is HA, it won't be affected by individual machine failures. Dragonfly doesn't seem to be HA.
+# Limitations
 
-## LAD from Facebook
+- If Docker registry throughput is not the bottleneck in your deployment workflow, switching to
+Kraken will not magically speed up your `docker pull`. To actually speed up `docker pull`, consider
+switching to [Makisu](https://github.com/uber/makisu) to improve layer reusability at build time, or
+tweak compression ratios, since `docker pull` spends most of the time on data decompression.
```
Comment (codygibb) [inline README.md]: s/since/as

File: README.md (hunk 1 above)
Comment (codygibb) [reply README.md]: s/It's/Kraken is

Comment (codygibb) [reply README.md]: Or maybe not, "some highlights of Kraken .... It's capable"

Maybe s/It's capable/Capable?

Comment (yiranwang52) [reply README.md]: `Kraken is capable` sounds better. I will change to that.

File: README.md (hunk 2 above)
Comment (yiranwang52) [reply README.md]: good catch.

File: README.md (hunk 3 above)
Comment (yiranwang52) [reply README.md]: ok

File: README.md (hunk 4 above)
Comment (yiranwang52) [reply README.md]: ok

**Pull request merged**